src/Dynamo.All.Net6.sln

```diff
@@ -122,6 +122,7 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GeometryColorTests", "..\test\Libraries\GeometryColorTests\GeometryColorTests.csproj", "{F8D2E3E5-4505-4463-9367-2EB2629D7DCD}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PackageManagerTests", "..\test\Libraries\PackageManagerTests\PackageManagerTests.csproj", "{2A70F3B5-7AEA-42FE-8DD5-8427C28D0025}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoAnalyzer", "DynamoAnalyzer\DynamoAnalyzer.csproj", "{ED4A6828-F72E-44DE-96E0-5F7E23DC6B10}"
```

Comment:
this is an overly general name - please make it more specific to the problem you are solving.

Reply:
renamed the project to 'DynamoPackagesAnalyzer'